storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -0,0 +1,169 @@
+
+**Self-signed certificates cannot be used with Edge Services, all certificates must be signed by a CA.**
+
+To get an SSL/TLS certificate for your domain or subdomain, you need to generate a Certificate Signing Request (CSR) and submit it to a Certificate Authority (CA) for them to validate your domain, who then send you a signed certificate. You may be able to carry out this procedure via your hosting provider, or from the command line.
+
+Find below some useful guides from different CAs on how to obtain a certificate from them:
+
+- [Let's Encrypt: Getting Started](https://letsencrypt.org/getting-started/). Let's Encrypt is a popular, non-profit CA that provides certificates at no cost. Note that the certificates are limited to 90 day validity, and must be renewed (again, at no cost) after this period.
+- [ZeroSSL: Creating an SSL Certificates](https://help.zerossl.com/hc/en-us/articles/360060119373-Creating-an-SSL-Certificate). ZeroSSL is another CA, which provides 90-day certificates at no cost, or certificates of longer duration at a price.
+- [Sectigo: Compare Sectigo SSL Certificates](https://www.sectigo.com/ssl-certificates-tls/compare). Sectigo is a CA who provide paid-for certificates with a validity period of between 1 and 6 years.
+
+## Accepted certificate types
+
+Types of validation:
+
+- ❌ **Self-signed certificates** (see above)
+- ✅ **Domain Validated Certificate**. The CA simply checks that the applicant owns the domain.
+- ✅ **Extended/Organization Validation Certificate**. The applicant must pass more in-depth validation procedures and checks by the CA.
+
+Types of domain coverage:
+
+- ✅ **Single domain certificate**. Secures a single domain or subdomain. Note that the certificate must be for `your-sub.domain.com`, where the subdomain corresponds to the [subdomain for Edge Services](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain). A single domain certificate simply for `yourdomain.com` would not be acceptable, as it would not cover the subdomain for Edge Services.
+- ✅ **Wildcard certificate**. Secures multiple subdomains for a domain, using a wildcard `*` symbol. The **Common Name** of the certificate should look like `*.yourdomain.com`.
+- ✅ **Multi-domain (MD) / Subject Alternative Name (SAN) / Unified Communications Certificate (UCC) certificate**.  Secures multiple explicitly-defined fully qualified domain names (`www.yourfirstdomain.com`, `sub.yourfirstdomain.com`, `yourfirstdomain.com`, `yourseconddomain.com`, `sub.yourseconddomain.com` etc.)
+
+## PEM format certificate chain
+
+Edge Services requires that you import your certificate as a `PEM`-formatted certificate chain, which includes the private key. PEM format is Base64 encoded ASCII, and by definition includes lines stating `-----BEGIN x-----` and `-----END x-----`.
+
+Your PEM formatted certificate chain should look like this:
+
+```
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@RoRoJ I think you can remove RSA, as other types of private keys may not have RSA in the section title.